client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx

```diff
@@ -14,13 +14,11 @@ export default localize( ( { translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="flag"
-				title={ translate( 'Backups & Security' ) }
+				title={ translate( 'Site Security' ) }
 				description={ translate(
-					'VaultPress makes it easy to keep an up-to-date backup of your site with both daily and real-time syncing of all ' +
-					'your WordPress content. To ensure your site stays safe, VaultPress performs security scans daily and makes it ' +
-					'easy to review and fix threats.'
+					'Your site is being securely backed up and scanned with real-time sync.'
 				) }
-				buttonText={ translate( 'View your backups' ) }
+				buttonText={ translate( 'Visit Security Dashboard' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Visit Dashboard' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).